src/playground/index.ejs

```diff
@@ -16,5 +16,8 @@
     <% } %>
   </head>
   <body>
+      <script>
+          window.StartLoad = (window.performance || Date).now();
```

Comment:
Looks like this was left in but isn't used in the code?

Reply:
Indeed. 😅